packages/editor-ui/src/components/Node/NodeCreator/composables/useActions.ts

```diff
@@ -163,6 +163,20 @@ export const useActions = () => {
 		};
 	}
 
+	/**
+	 * Checks if added nodes contain trigger followed by another node
+	 * In this case, we should connect the trigger with the following node
+	 */
+	function shouldConnectWithExistingTrigger(addedNodes: AddedNode[]): boolean {
+		const { selectedView } = useNodeCreatorStore();
+		if (selectedView === TRIGGER_NODE_CREATOR_VIEW && addedNodes.length === 2) {
```

Comment:
Do we need to check for the `TRIGGER_NODE_CREATOR_VIEW` here? What if user is at the regular view but added the node by searching for it(which would show nodes from both views)?

Reply:
In that case we don't need it. I wasn't sure about all of the options to add nodes but decided to go conservative with the fix. Will remove